spec/models/refinery/jobs/job_spec.rb

```diff
@@ -16,6 +22,40 @@ module Jobs
           expect(FactoryGirl.build(:job, {description: 1, title: 1})).to be_valid
         end
       end
+
+      context 'update' do
+        it 'should be functional' do
+          gen_title = generate_string
+          gen_description = generate_string
+
+          @job.update_attributes({title: gen_title, description: gen_description })
```

Comment:
Line is too long. [83/80]<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.